.travis.yml

```diff
@@ -70,13 +68,6 @@ jobs:
         chrome: stable
         firefox: latest
       script: scripts/raven-js.sh
-    - stage: Saucelabs
```

Comment:
Why?

Reply:
Because they are 100% broken, need to invest time to fix.